src/app/selectors.ts

```diff
@@ -206,3 +206,6 @@ export const inviterAddressSelector = (state: RootState) => state.app.inviterAdd
 export const networkTimeoutSecondsSelector = (state: RootState) => state.app.networkTimeoutSeconds
 
 export const celoNewsConfigSelector = (state: RootState) => state.app.celoNews
+
+export const useTwelveWordSeedPhraseSelector = (state: RootState) =>
+  state.app.useTwelveWordSeedPhrase
```

Comment:
so it doesn't look like a hook:
```suggestion
export const twelveWordSeedPhraseEnabledSelector = (state: RootState) =>
  state.app.twelveWordSeedPhraseEnabled
```

Reply:
Done in b1a5a1d4d9a4e90b601b5797f8e6c32cd78b30c9!